README.md

```diff
@@ -205,6 +206,18 @@ the required file format at step up time, the job can be configured properly. Fo
 3. User wants to restore that masking job to another engine. They can achieve this as long as
  `my_file_format.txt`  is next to the backup file, or in the backup folder.
 
+# JDBC Drivers
+
+Unfortunately the masking engine does not provide a way to retrieve JDBC driver files that have been uploaded,
```

Comment:
Currently, I will prefer the API endpoints to back up/ restore all drivers as API endpoints are already tested. Sync functionality for back up/ restore all drivers is still in development and also need to test properly. But yes we can change it to sync later on once it will be available.